NOTICE

```diff
@@ -77,4 +77,14 @@ Copyright (2023) Databricks, Inc.
 https://github.com/databrickslabs/blueprint/blob/main/LICENSE
 
 Databricks Connect - https://pypi.org/project/databricks-connect/
-Copyright (2019) Databricks Inc.
\ No newline at end of file
+Copyright (2019) Databricks Inc.
+
+circe - https://github.com/circe/circe
+Copyright (c) 2015, Ephox Pty Ltd, Mark Hibberd, Sean Parsons, Travis Brown, and other contributors. All rights reserved.
+https://github.com/circe/circe/blob/series/0.14.x/LICENSE
```

Comment:
this is Apache2 license, move it away from Databricks License section to a respective Apache2 section, right after `cryptography`, for example.